guidelines/monai-application-package.md

```diff
@@ -1,498 +1,650 @@
+| `resources.gpuMemoryLimit`                              | No       | N/A                     | string      | memory size               | The GPU memory limit for the Application or the Fragment. (1)                                                 |
+| `resources.sharedMemory`                                | No       | `64Mi`                  | string      | memory size               | The shared memory required by the Application or the Fragment.                                                |
+| `resources.fragments`                                   | No       | N/A                     | object      | objects                   | Nested objects which describe resources for a Multi-Fragment Application.                                     |
+| `resources.fragments.<fragment-name>`                   | **Yes**  | N/A                     | string      | string                    | Name of the fragment.                                                                                         |
+| `resources.fragments.<fragment-name>.cpu`               | No       | `1`                     | decimal (2) | number                    | Number of CPU cores required by the Fragment.                                                                 |
+| `resources.fragments.<fragment-name>.cpuLimit`          | No       | N/A                     | decimal (2) | number                    | The CPU core limit for the Fragment. (1)                                                                      |
+| `resources.fragments.<fragment-name>.gpu`               | No       | `0`                     | decimal (2) | number                    | Number of GPU devices required by the Fragment.                                                               |
+| `resources.fragments.<fragment-name>.gpuLimit`          | No       | N/A                     | decimal (2) | number                    | The GPU device limit for the Fragment. (1)                                                                    |
+| `resources.fragments.<fragment-name>.memory`            | No       | `1Gi`                   | string      | memory size               | The memory required by the Fragment.                                                                          |
+| `resources.fragments.<fragment-name>.memoryLimit`       | No       | N/A                     | string      | memory size               | The memory limit for the Fragment. (1)                                                                        |
+| `resources.fragments.<fragment-name>.gpuMemory`         | No       | `1Gi`                   | string      | memory size               | The GPU memory required by the Fragment.                                                                      |
```

Comment:
Is `1Gi` the correct default value for `resources.fragments.<fragment-name>.gpuMemory` if the default value for `resources.fragments.<fragment-name>.gpu` is `0`

Reply:
Updating to `N/A`